src/paperless_mail/admin.py

```diff
@@ -119,6 +119,10 @@ class MailRuleAdmin(GuardedModelAdmin):
 
     ordering = ["order"]
 
+    raw_id_fields = ("assign_correspondent",)
```

Comment:
```suggestion
    raw_id_fields = ("assign_correspondent", "assign_document_type")
```

Any reason to only include correspondent assignment here not `assign_document_type`? I suppose for `assign_tags` the UI is so much worse maybe not 'worth it'.

Reply:
For `assign_tags` we already use `filter_horizontal` which should be good enough. 
Regarding `assign_document_type` you are right, we can include it. I initially intentionally left this out because I don't think that users use hundreds or thousands of document types, but I could be wrong.

Reply:
Is there any hit to including it just in case?

Reply:
Let's add it.